foodsaving/invitations/api.py

```diff
@@ -66,3 +67,30 @@ def accept(self, request, **kwargs):
         serializer.is_valid(raise_exception=True)
         serializer.save()
         return Response(serializer.data)
+
+
+class InvitationResendEmailViewSet(GenericViewSet):
+    queryset = Invitation.objects
+    serializer_class = InvitationResendEmailSerializer
+    permission_classes = (
+        IsAuthenticated,
+    )
+    lookup_field = 'pk'
+
+    def get_queryset(self):
+        return self.queryset.filter(expires_at__gte=timezone.now())
+
+    @action(detail=True, methods=["POST"])
+    def resend_invitation_email(self, request, **kwargs):
```

Comment:
Can you move this action into the `InvitationsViewSet` and rename it to `resend`? 

I think it's useful if both share the same queryset, then we don't need to replicate the filtering. You also don't need to use an extra serializer (see other comment).

Reply:
Yeah, sounds better, I've tried this approach but I got a little bit lost with the permissions, but after seeing the examples, I think I can do it.